### ISSUE TITLE
3-6, 5-7 recognized as erroneous

### ISSUE THREAD

**Issue by cpg** [racketlogger/tennis#1]
this is a bug. it's a perfectly valid score for the away player. it's kind of urgent.

```
irb(main):001:0> ts = Tennis.new('3-6, 5-7')
=> #<Tennis:0x007f89208c3f88 @winner=:error, @scores=[3, 6, 5, 7], @points=[0, 0], @sets_lost=[2, 0], @games_lost=[13, 8]>
irb(main):002:0> ts.winner
=> :error
irb(main):003:0>
```

**Comment by cpg** [comment]
Please write a test for this kind of corner case.

**Comment by rohankatyal29** [comment]
It's fixed: b3a716c3f1b6ab49b9a1bb8648fc82c1add9b118 
I have added a test case for this as well.

**Comment by rohankatyal29** [comment]
It's fixed: b3a716c3f1b6ab49b9a1bb8648fc82c1add9b118 
I have added a test case for this as well.
